xamarin/Xamarin.Forms
# [Shell, Android, iOS] shell menu scrolling

### Description of Change ###

Added `FlyoutVerticalScrollMode` property. Which when set to `Disabled` disables vertically scroll of menu. `Auto` setting enables scrolling if items do not fit into the flyout menu.

### Issues Resolved ### 

- fixes #4765

### API Changes ###

Added:
 - ScrollMode enum;
 - ScrollMode Shell.FlyoutVerticalScrollMode { get; set; } //Bindable Property

### Platforms Affected ### 

- Core/XAML (all platforms)
- iOS
- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 



### Testing Procedure ###

- run ui test 4756
- scroll of flyout menu should not work
- change Scroll Mode in picker
- scroll of flyout menu should work 
if Scroll Mode is Enabled
if Scroll Mode is Auto and not all items is shown
- scroll of flyout menu shouldn't work 
if Scroll Mode is Disabled
if Scroll Mode is Auto and all items is shown

### PR Checklist ###
